src/components/Dropdown/SubtleDropdown.tsx

```diff
@@ -22,7 +23,8 @@ export function SubtleDropdown({
   items = {},
   name,
   onChange = () => {},
-  value
+  value,
+  checkStatus
```

Comment:
checkStatus should not be a field on SubtleDropdown - SubtleDropdown is a generic component it can be disabled/hidden or have different modes of operation but it should have any fields that are specific to 1 use case.

Reply:
@moshloop I used disabled instead of checkStatus in the latest commit. I think it can be applied to any other components using the Dropdown. Please do check. Thanks